build-npm-module/index300.js

```diff
@@ -0,0 +1,7 @@
+const caseConverter = require('./index');
+
+const str = 'hello free Code Camp!';
+
+// Let's try your getUpperCase function. Conver `str` to upper case and print it.
```

Comment:
This could use a quick example to show how to call the `getUpperCase` function:

Let's try your `getUpperCase()` function.
Here's an example that imports a module, calls a function, and prints the result to the console:
```
const arrLengthifier = require('arr_lengthifier');
const arr = [1, 2, 3];

console.log(arrLengthifier.getArrLength(arr));
```
Convert `str` to upper case and print it to the console.